test/filters/browser_tests.filter

```diff
@@ -967,6 +967,11 @@
 # optimization_guide::features::kRemoteOptimizationGuideFetching
 -ParametrizedTests/PreloadingModelKeyedServiceTest.FeatureFlagIsWorking/*
 
+# These test fail because we remove description and 3p cookies toggle from the
+# Page Info cookies view.
```

Comment:
it would be nice if we had a list of tests that we expect to fail and "success" would mean that all of them fail. That way we could track that these are indeed removed without creating our own tests (which we should otherwise create to verify our overrides are working)